documentation/source/newsfragments/1705.bugfix.1.rst

```diff
@@ -0,0 +1 @@
+Scheduler now correctly cleans up when a test is failed using the ``$fatal()`` System Task in Verilog.
```

Comment:
```suggestion
The scheduler now correctly cleans up when a test is failed using the ``$fatal()`` System Task in Verilog.
```

Reply:
Is this actually a bug that was present in 1.3? Or just a bug that was introduced by the commit before it?

Reply:
The latter should not be in a release note.

Reply:
@eric-wieser I guess that's true, it worked before. I'll remove the newsfragment.